device/arista/x86_64-arista_7060x6_64pe_b/Arista-7060X6-64PE-B-C448O16/th5-a7060x6-64pe.config.bcm

```diff
@@ -3361,7 +3361,7 @@ bcm_device:
     0:
         global:
             ftem_mem_entries: 65536
-            sai_stats_support_mask: 0x80
+            sai_stats_support_mask: 0x480
```

Comment:
Why these counters are removed from port?

Reply:
It's to mask out PFC pause frame from drop counter.